pilot/pkg/model/auth_cache.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2017 Istio Authors
```

Comment:
name the file jwks_resolver to match with the class and functionality in the file.

Reply:
Also, it's been 2018 for a while now :-)

Reply:
thanks for the reminder :)